src/OpenTelemetry.Exporter.NewRelic/NewRelicTraceExporter.cs

```diff
@@ -82,44 +84,32 @@ internal NewRelicTraceExporter(SpanDataSender spanDataSender, TelemetryConfigura
             }
         }
 
-
-        /// <summary>
-        /// Responsible for sending Open Telemetry Spans to New Relic endpoint.
-        /// </summary>
-        /// <param name="batch">Collection of Open Telemetry spans to be sent to New Relic</param>
-        /// <param name="cancellationToken"></param>
-        /// <returns></returns>
-        public override async Task<ExportResult> ExportAsync(IEnumerable<Activity> batch, CancellationToken cancellationToken)
+        /// <inheritdoc />
+        public override ExportResult Export(in Batch<Activity> batch)
         {
-            if (batch == null) return ExportResult.Success;
-
             var nrSpanBatch = ToNewRelicSpanBatch(batch);
 
-            if(nrSpanBatch.Spans.Count == 0)
+            if (nrSpanBatch.Spans.Count == 0)
             {
                 return ExportResult.Success;
             }
 
-            var result = await _spanDataSender.SendDataAsync(nrSpanBatch);
+            Response response = null;
+            Task.Run(async () => response = await _spanDataSender.SendDataAsync(nrSpanBatch)).GetAwaiter().GetResult();
 
-            switch (result.ResponseStatus)
+            switch (response.ResponseStatus)
             {
                 case NewRelicResponseStatus.DidNotSend_NoData:
                 case NewRelicResponseStatus.Success:
                     return ExportResult.Success;
                
                 case NewRelicResponseStatus.Failure:
                 default:
-                    return ExportResult.FailedRetryable;
+                    return ExportResult.Failure;
```

Comment:
Just double checking.   We can do this b/c the Telemetry SDK is responsible for doing retry logic?  If the execution path arrives here, it would be b/c Telemetry SDK already exhausted its retries?

Reply:
Telemetry SDK never had retry logic. There may have been thoughts/plans in the past, but the spec has since change to only include Success and Failure.